### PR TITLE
docs updates and fixes

### PR DESCRIPTION
**Pull request by sfc-gh-pmardziel**
# Description

- Added release policies readme.

- Fixed many docs issues especially around the API reference pages.

## Other details good to know for developers

Please include any other details of this change useful for _TruLens_ developers.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
  not work as expected)
- [ ] New Tests
- [ ] This change includes re-generated golden test results
- [x] This change requires a documentation update

### CODE REVIEW

**Review by sfc-gh-jreini — APPROVED** [review]

File: POLICIES.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - Warning period: 2024-09-01 (`trulens-eval==1.0.0a0`) to
+    2024-10-14. Backwards compatibility during the warning period is provided by
+    the new content of the `trulens_eval` package which provides aliases to the
+    features in their new locations. See
+    [trulens_eval](trulens/api/trulens_eval/index.md).
+
+    - Deprecated period: 2024-10-14 to 2025-12-01. Usage of `trulens_eval` will
+  	produce errors indicating deprecation.
+
+    - Removed expected 2024-02-01 Installation of the latest version of
```
Comment (sfc-gh-chu) [inline POLICIES.md:48]: Is this supposed to be 2024-12-01?

File: POLICIES.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+in the version numbering. In such cases, the appropriate level version change
+will occur at the introduction of the warning period.
+
+## Currently deprecating features
+
+- Starting 1.0.0, the `trulens_eval` package is being deprecated in favor of
+  `trulens` and several associated required and optional packages. See
+  [trulens_eval migration](/trulens/guides/trulens_eval_migration) for details.
+
+    - Warning period: 2024-09-01 (`trulens-eval==1.0.0a0`) to
```
Comment (sfc-gh-chu) [inline POLICIES.md:39]: we probably don't need to reference prerelease version and would likely be 1.0.1 instead

File: docs/reference/index.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+- `trulens_eval` installs [trulens_eval](trulens_eval), a temporary package for backwards compatibility.
+
+Two categories of optional packages contain integrations with 3rd party app
+types and providers:
+
+- [Apps](apps/index.md).
+
+    - [TruChain][trulens.apps.langchain.TruChain] in package
+        `trulens-app-langchain`.
```
Comment (sfc-gh-chu) [inline docs/reference/index.md:24]: nit: trulens-apps-langchain

same for others in this file

File: POLICIES.md (hunk 1 above)
Comment (sfc-gh-pmardziel) [reply POLICIES.md:48]: yes, fixing in another PR

File: POLICIES.md (hunk 2 above)
Comment (sfc-gh-pmardziel) [reply POLICIES.md:39]: ok; fixing in another PR.

File: docs/reference/index.md (hunk 3 above)
Comment (sfc-gh-pmardziel) [reply docs/reference/index.md:24]: thx, fixing in another PR

**Pull request merged**